stable/cluster-autoscaler/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 3.2.0
+version: 3.2.1
```

Please bump the minor version, as this adds a feature.

```suggestion
version: 3.3.0
```

---

updated!